### ISSUE TITLE
Pull/grab system needs to change

### ISSUE THREAD

**Issue by IzzaBinS33n** [tgstation/TerraGov-Marine-Corps#617]
The current pull/grab system is absolute ASS...it takes up a slot to pull something and it stunlocks marines from being able to break free of their alien predecessors. It also stun blocks marines from breaking free from other marines when they grab them.  I would prefer to use the /tg/ pull system where you have to *grab* your target so they are unable to break free of your clutches... now of course this would slightly nerf xenos on pulling their prey into the hive, BUT (and this is a big but) I suggest that the time it takes for the grab intent to go to *grabs aggressively* should be halved for xenos since they are much bigger and nimbler. As well as the lower the health of the marine the easier they are to grab aggressively. This pull passively becomes an instant intensive grab and it really doesn't make sense either. For example: When you go to grab someone aggressively it takes awhile since they are resisting and to immediately pull them would take some time and would be slow as hell. My proposition is to remove this grab/pull system implemented and make it more user friendly so that it doesn't clog up a slot in their hand.
I would also recommend changing this based on reasons to make coding more loose and maybe add a bit more interest into round. So if a xeno grabs a marine passively they can RUN at nearly the speed they go, but the marine can break away if they are standing or get up. This would make it so that marines aren't just rag dolls to the xenomorphs and easily just get slashed once or twice in a pull and BAM cant move anymore. If this system were implemented the Xenos would not be nerfed but rather buffed. They would be buffed since they are able to grab marines faster as well as make them like true xenos that attack hard knocking their opponent down in a few hits and then run off with them, unlike this where they just waltz in with thicc ass armor and grab someone and no one can get him/her back even with combined fire. What is currently here is the Xenos spam the pull button as much as they can because it is the only way they can really kill an ass load marines and this is depressing. This really reminds me of those games where you just spam a single button against the AI and not even the game can fight you back. If this system is implemented you can start working on the more added mechanics to the game that really need attention. I personally would not mind coding this in...but i make this a thread in order to obtain some sort of consensus and maybe even a debate that could change TGMC entirely. Your medical is great. your xenomorph cool down and abilities are nearly balanced with the marines weaponry of power. But the core mechanics that constrain this game need to be let loose so that the player is able to enjoy their game. The reason i am doing this is (slightly salty but that is not the main point) to fix up the loose ends CM decided to constrict onto it in order to make it their *unique code*. As I've seen you have made things a lot easier for the player and removed features that really are unnecessary and added those that are both realistic and really fucking cool!
 In conclusion my intent of this *poll* is to spread the message of bringing this code into the eyes of the many not the eyes of the few. I am not certain what your response will be or if your willing to accept a change in this *pull/grab* system implemented in the code. I hope you at least take the time to read and understand my intentions and help make this great server you have bestowed upon us be able to enjoy a realistic game of Marines fighting Aliens.

P.S im writing this late at night so don't expect it to be perfect and I've read this thing 50 times over now re modifying every word so if anything you want clarification on please message me i will talk as soon as i can.

Sincerly, IzzaBinSeen

**Comment by Rohesie** [comment]
This is where you'll want to be posting this: https://tgstation13.org/phpBB/viewforum.php?f=65
